modules/core/src/main/java/org/apache/ignite/internal/commandline/CommandList.java

```diff
@@ -62,7 +63,10 @@
     READ_ONLY_ENABLE("--read-only-on", new ClusterReadOnlyModeEnableCommand()),
 
     /** */
-    READ_ONLY_DISABLE("--read-only-off", new ClusterReadOnlyModeDisableCommand());
+    READ_ONLY_DISABLE("--read-only-off", new ClusterReadOnlyModeDisableCommand()),
+
+    /** */
+    MANAGEMENT("--management", new ManagementCommands());
```

Comment:
@agoncharuk I think we need to hide this new command same as we do for 'rolling upgrade and baseline autoadjustment'. Agree?